oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/SampleFamily.java

```diff
@@ -575,4 +588,31 @@ private static boolean doubleComp(CompType compType, double a, double b) {
     private enum CompType {
         EQUAL, NOT_EQUAL, LESS, LESS_EQUAL, GREATER, GREATER_EQUAL
     }
+
+    private interface Retag {
+        Sample[] execute(Sample[] ss, String newLabelName, String existingLabelName);
+    }
+
+    protected enum K8sRetagType implements Retag {
```

Comment:
Let's move this into another file. SampleFamily is big enough.

Reply:
done